src/main/java/wolox/training/exceptions/RestExceptionController.java

```diff
@@ -0,0 +1,30 @@
+package wolox.training.exceptions;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class RestExceptionController  extends ResponseEntityExceptionHandler {
```

Comment:
you can to manage the class names according his function , can rename this class like RestExceptionHandler

Reply:
It was solve last commit